_data/communication.yml

```
@@ -216,6 +216,13 @@ websites:
     img: line.png
     facebook: line.worldwide
 
+  - name: Litmus
+    url: https://www.litmus.com
+    img: litmus.svg
+    tfa:
+      - sms
```

```suggestion
      - sms
      - proprietary
```
According to the site:

> Is there an alternative to SMS-based verification? You can use Authy for our two-step verification. If you download and install the Authy apps you will need to enter the same phone number you used for your Litmus SMS-based two-step verification.
> 
> Once your Authy account is set up, when you're prompted for your two-step verification code, you can use either the code sent by SMS or the code generated by Authy.